index.js

```diff
@@ -123,19 +123,20 @@ CSSModules.prototype.fetchExports = function(importPath, fromFile) {
 
 CSSModules.prototype.loadPath = function(dependency) {
   var seen = this._seen;
+  var encoding = this.encoding;
+
   var absolutePath = dependency.toString();
-  if (seen[absolutePath]) {
-    return Promise.resolve(seen[absolutePath]);
+  var loadPromise = seen[absolutePath];
+
+  if (!loadPromise) {
+    loadPromise = new Promise(function(resolve) {
+      var content = fs.readFileSync(absolutePath, this.encoding);
```

Comment:
If `this.encoding` is used here, declaring it as a variable above is not required.